examples/dynamic-config-cp/Dockerfile-control-plane

```diff
@@ -1,6 +1,5 @@
 FROM golang
 
-RUN apt-get -y update && apt-get install -y -qq --no-install-recommends netcat
```

Comment:
it turns out that this is required - its used in a `healthcheck` in the compose file

Reply:
No, it used in ES image I think

Reply:
here 

https://github.com/envoyproxy/envoy/blob/19210537c7169ed7c8560cdaf16f9b3590db3215/examples/dynamic-config-cp/docker-compose.yaml#L28-L29

Reply:
Oh I miss it.